ResearchKit/Common/ORKTextFieldView.h

```diff
@@ -35,6 +35,11 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+static NSString* const kEmptyBullet = @"\u25CB";
+static NSString* const kFilledBullet = @"\u25CF";
+static NSString* const k4DigitPin = @"\u25CB\u25CB\u25CB\u25CB";
+static NSString* const k6DigitPin = @"\u25CB\u25CB\u25CB\u25CB\u25CB\u25CB";
```

Comment:
If these strings are only referenced by `ORKPasscodeStepViewController`, then move them to `ORKPasscodeStepViewController.m`